app/containers/TextInput/FormTextInput.tsx

```diff
@@ -28,8 +28,8 @@ const styles = StyleSheet.create({
 		fontSize: 16,
 		paddingHorizontal: 16,
 		paddingVertical: 10,
-		borderWidth: 1,
-		borderRadius: 4
+		borderWidth: 2,
+		borderRadius: 2
```

Comment:
We changed this a while ago intentionally.
Feels weird/old to have such a thick border width.
Can you create a thread with Jose about it and mention me? 😄